src/applications/disability-benefits/4142/config/form.js

```diff
@@ -0,0 +1,252 @@
+// _ from 'lodash/fp';
+
+// Example of an imported schema:
+// import fullSchema from '../22-4142-schema.json';
+// In a real app this would be imported from `vets-json-schema`:
+// import fullSchema from 'vets-json-schema/dist/22-4142-schema.json';
+
+// In a real app this would not be imported directly; instead the schema you
+// imported above would import and use these common definitions:
+import commonDefinitions from 'vets-json-schema/dist/definitions.json';
```

Comment:
Is this a temporary solution?

It's probably okay for now, but these definitions should be imported into the 4142 schema and used from there. Importing all the definitions will make the payload to the client bigger than it needs to be.

Reply:
I will pull in 4142 schema locally. It will be the temporary solution until schema is merged into master.